AutomatedTesting/Gem/PythonTests/scripting/VariableManager_Default_CreateDeleteVars.py

```diff
@@ -30,81 +38,83 @@ def VariableManager_Default_CreateDeleteVars():
-        add_button.click()  # Click on Create Variable button
-        # Select variable type
-        table_view = variable_manager.findChild(QtWidgets.QTableView, "variablePalette")
-        model_index = pyside_utils.find_child_by_pattern(table_view, var_type)
-        # Click on it to create variable
-        pyside_utils.item_view_index_mouse_click(table_view, model_index)
-        # Verify if the variable is created
-        # NOTE: To check if variable of a type is created, we are checking 1) rowcount
-        # 2) If we have row with variable "Variable <index>"
-        # 3) Type of variable, which is next column of the variable name
-        result = graph_vars.model().rowCount(QtCore.QModelIndex()) == (
-            index + 1
-        )  # since we added 1 variable, rowcount will increase by 1
-        var_mi = pyside_utils.find_child_by_pattern(graph_vars, f"Variable {index+1}")
-        result = result and (var_mi is not None) and (var_mi.siblingAtColumn(1).data(Qt.DisplayRole) == var_type)
-        Report.result(generate_test_tuple(var_type, "create"), result)
-
-    # 6) Delete each type of variable and verify if it is deleted
-    for index, var_type in enumerate(var_types):
-        # Delete variable and verify if its deleted
-        # NOTE: To check if variable of a type is deleted, we are checking rowcount
-        var_mi = pyside_utils.find_child_by_pattern(graph_vars, f"Variable {index+1}")
-        pyside_utils.item_view_index_mouse_click(graph_vars, var_mi)
-        QtTest.QTest.keyClick(graph_vars, Qt.Key_Delete, Qt.NoModifier)
-        # since variable is deleted, rowcount will decrease by 1
-        result = graph_vars.model().rowCount(QtCore.QModelIndex()) == (len(var_types) - (index + 1))
-        Report.result(generate_test_tuple(var_type, "delete"), result)
-
-    # 7) Close SC window
-    general.close_pane("Script Canvas")
-
-
-if __name__ == "__main__":
-    import ImportPathHelper as imports
-
-    imports.init()
-    from utils import Report
-
-    Report.start_test(VariableManager_Default_CreateDeleteVars)
+
+        graph_vars = variable_manager.findChild(QtWidgets.QTableView, "graphVariables")
+
+        # 5) Create variable_type of each type and verify if it is created
+        for index, variable_type in enumerate(VARIABLE_TYPES):
+            # Create new variable_type
+            add_button = variable_manager.findChild(QtWidgets.QPushButton, "addButton")
+            # Click on Create Variable button and wait for it to render
+            add_button.click()
+            helper.wait_for_condition((
+                lambda: variable_manager.findChild(QtWidgets.QTableView, VARIABLE_PALETTE_QT) is not None), WAIT_TIME_3)
+
+            # Select variable_type type
+            table_view = variable_manager.findChild(QtWidgets.QTableView, VARIABLE_PALETTE_QT)
+            model_index = pyside_utils.find_child_by_pattern(table_view, variable_type)
```

Comment:
This comment doesn't seem to do what the code appears to be doing.

This appears to be building data to be used elsewhere instead of doing any actual selections.